genesis/process/genesisBlockCreator_test.go

```diff
@@ -467,7 +439,7 @@ func TestNewGenesisBlockCreator(t *testing.T) {
 		t.Parallel()
 
 		arg := createMockArgument(t, "testdata/genesisTest1.json", &mock.InitialNodesHandlerStub{}, big.NewInt(22000))
-		rtComponents := mainFactoryMocks.NewRunTypeComponentsStub()
+		rtComponents := genesisMocks.NewRunTypeComponentsStub()
```

Comment:
Please revert and use existing stubs.
We already have too many duplicated ones

Reply:
cannot use existing stub because of cyclic imports